src/App/App.jsx

```diff
@@ -0,0 +1,121 @@
+import CountdownScreen from '../pages/CountdownScreen'
+import StartScreen from '../pages/StartScreen'
+import { useState } from 'react'
+import { Route, Switch, useHistory } from 'react-router-dom'
+
+function App() {
+  const { push } = useHistory()
+  const DURATIONTWENTYFIVE = { minutes: 25, seconds: 0 }
```

Comment:
Maybe consider to use Camel case for instance `durationTwentyFive` to name your variables. This would be much easier to read.

Reply:
Thanks a lot for your review! Really appreciate it! 🙏

According to ESLint, hard-coded config values have to be written in capital letters :)
You can take a look at the example [here](https://eslint.org/docs/rules/no-magic-numbers)

Reply:
Alright, I learned something new 😉
Thx for sharing.